ui/app/AppLayouts/Chat/views/communities/CommunityTokenView.qml

```diff
@@ -231,11 +252,20 @@ StatusScrollView {
             }
 
             CustomPreviewBox {
+                id: decimalsBox
+
+                visible: root.isAssetView
+                label: qsTr("DP")
```

Comment:
What is `DP`? :thinking:

Reply:
It represents the `decimals` field but.. Yeah.. I will ask 🤔

Reply:
@caybro --> `Decimal Points`! :)

Reply:
For me, it's "Display Pixels" ;) It deserves some better string imho, wdyt @benjthayer ?

Reply:
> For me, it's "Display Pixels" ;) It deserves some better string imho, wdyt @benjthayer ?

Yeah its not the ideal label - it was done for a couple of reasons:
1) this will always be 1 or 2 digits (max decimals is 10)
2) to give the row of data the best chance of not wrapping to 2 lines

Perhaps we can amend the preceding form label to `Decimals (DP)` so when the user encounters the abbreviated form, they can more easily map the meaning back to the form?

WDYT @John-44

Reply:
> > For me, it's "Display Pixels" ;) It deserves some better string imho, wdyt @benjthayer ?
> 
> Yeah its not the ideal label - it was done for a couple of reasons:
> 
> 1. this will always be 1 or 2 digits (max decimals is 10)
> 2. to give the row of data the best chance of not wrapping to 2 lines
> 
> Perhaps we can amend the preceding form label to `Decimals (DP)` so when the user encounters the abbreviated form, they can more easily map the meaning back to the form?
> 
> WDYT @John-44

@benjthayer sounds good to me!  thx :-)

Reply:
@caybro that is now updated in the Figma:

'Decimals' field re-labelled to 'Decimals (DP)'

https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?type=design&node-id=29373-560486&t=zPDsmAx9Adui8iUF-4